x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer_embeddable_chart/timeseriesexplorer_embeddable_chart.js

```diff
@@ -32,13 +32,16 @@ import {
 } from '@elastic/eui';
 import { TimeSeriesExplorerHelpPopover } from '../timeseriesexplorer_help_popover';
 
+import { ANOMALIES_TABLE_DEFAULT_QUERY_SIZE } from '../../../../common/constants/search';
 import {
   isModelPlotEnabled,
   isModelPlotChartableForDetector,
   isSourceDataChartableForDetector,
 } from '../../../../common/util/job_utils';
 
 import { LoadingIndicator } from '../../components/loading_indicator/loading_indicator';
+import { ml } from '../../services/ml_api_service';
```

Comment:
`mlApiServices` is available in the context here via `this.context.services.mlService` so we shouldn't need to import it.

Reply:
Updated in [70ab770](https://github.com/elastic/kibana/pull/176788/commits/70ab770cc5e06985dba0dd7b86123eee21f7a72e).